justfile

```diff
@@ -23,7 +24,12 @@ build-deploy-cloudbeat-debug: build-cloudbeat-debug load-cloudbeat-image deploy-
 load-cloudbeat-image:
   kind load docker-image cloudbeat:latest --name kind-mono
 
+build-opa-bundle:
+  CSP_POLICIES_PKG_DIR=$( go list -m -f {{{{.Dir}} {{cspPoliciesPkg}} ) && \
+  opa build -b ${CSP_POLICIES_PKG_DIR}/bundle -e ${CSP_POLICIES_PKG_DIR}/bundle/compliance
```

Comment:
Does that assume I have the opa binary locally?

Reply:
yeap, we added hermit to the repo. So opa can be added to it.
https://github.com/elastic/cloudbeat/pull/380#discussion_r971697163